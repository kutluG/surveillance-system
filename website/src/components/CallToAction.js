import React from 'react';
import { motion } from 'framer-motion';
import { Link } from 'react-router-dom';

export default function CallToAction() {
  return (
    <section className="py-24 bg-gradient-to-br from-slate-900 via-blue-900 to-slate-900 relative overflow-hidden">
      {/* Background Effects */}
      <div className="absolute inset-0">
        <div className="absolute top-1/4 left-1/4 w-96 h-96 bg-blue-500/10 rounded-full blur-3xl animate-pulse-slow"></div>
        <div className="absolute bottom-1/4 right-1/4 w-96 h-96 bg-cyan-500/10 rounded-full blur-3xl animate-pulse-slow delay-1000"></div>
      </div>

      <div className="container mx-auto px-6 relative z-10">
        <motion.div
          initial={{ opacity: 0, y: 50 }}
          whileInView={{ opacity: 1, y: 0 }}
          transition={{ duration: 0.8 }}
          viewport={{ once: true }}
          className="text-center max-w-4xl mx-auto"
        >
          <h2 className="text-4xl md:text-5xl lg:text-6xl font-bold text-white mb-6">
            Ready to Secure Your
            <span className="block text-transparent bg-clip-text bg-gradient-to-r from-blue-400 to-cyan-400">
              Digital Perimeter?
            </span>
          </h2>
          
          <p className="text-xl text-slate-300 mb-12 leading-relaxed">
            Join hundreds of organizations already using our AI-powered surveillance system 
            to protect their assets with enterprise-grade security and real-time monitoring.
          </p>

          <div className="flex flex-col sm:flex-row gap-6 justify-center items-center">
            <motion.div
              whileHover={{ scale: 1.05 }}
              whileTap={{ scale: 0.95 }}
            >
              <Link
                to="/dashboard"
                className="inline-flex items-center px-8 py-4 bg-gradient-to-r from-blue-600 to-cyan-600 text-white font-semibold rounded-lg hover:from-blue-700 hover:to-cyan-700 transition-all duration-300 shadow-lg hover:shadow-2xl group"
              >
                <span>Start Free Trial</span>
                <svg className="w-5 h-5 ml-2 group-hover:translate-x-1 transition-transform" fill="none" stroke="currentColor" viewBox="0 0 24 24">
                  <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M13 7l5 5m0 0l-5 5m5-5H6" />
                </svg>
              </Link>
            </motion.div>

            <motion.div
              whileHover={{ scale: 1.05 }}
              whileTap={{ scale: 0.95 }}
            >
              <Link
                to="/documentation"
                className="inline-flex items-center px-8 py-4 border-2 border-slate-600 text-slate-300 font-semibold rounded-lg hover:border-blue-500 hover:text-white transition-all duration-300 group"
              >
                <span>View Documentation</span>
                <svg className="w-5 h-5 ml-2 group-hover:translate-x-1 transition-transform" fill="none" stroke="currentColor" viewBox="0 0 24 24">
                  <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M9 12h6m-6 4h6m2 5H7a2 2 0 01-2-2V5a2 2 0 012-2h5.586a1 1 0 01.707.293l5.414 5.414a1 1 0 01.293.707V19a2 2 0 01-2 2z" />
                </svg>
              </Link>
            </motion.div>
          </div>

          {/* Trust Indicators */}
          <motion.div
            initial={{ opacity: 0, y: 30 }}
            whileInView={{ opacity: 1, y: 0 }}
            transition={{ duration: 0.8, delay: 0.3 }}
            viewport={{ once: true }}
            className="mt-16 pt-8 border-t border-slate-700"
          >
            <p className="text-slate-400 text-sm mb-6">Trusted by industry leaders</p>
            <div className="flex flex-wrap justify-center items-center gap-8 opacity-60">
              <div className="flex items-center space-x-2">
                <div className="w-8 h-8 bg-slate-700 rounded"></div>
                <span className="text-slate-400 font-medium">TechCorp</span>
              </div>
              <div className="flex items-center space-x-2">
                <div className="w-8 h-8 bg-slate-700 rounded"></div>
                <span className="text-slate-400 font-medium">SecureInc</span>
              </div>
              <div className="flex items-center space-x-2">
                <div className="w-8 h-8 bg-slate-700 rounded"></div>
                <span className="text-slate-400 font-medium">DataGuard</span>
              </div>
              <div className="flex items-center space-x-2">
                <div className="w-8 h-8 bg-slate-700 rounded"></div>
                <span className="text-slate-400 font-medium">CloudShield</span>
              </div>
            </div>
          </motion.div>

          {/* Contact Info */}
          <motion.div
            initial={{ opacity: 0 }}
            whileInView={{ opacity: 1 }}
            transition={{ duration: 0.8, delay: 0.5 }}
            viewport={{ once: true }}
            className="mt-12 text-center"
          >
            <p className="text-slate-400 text-sm">
              Need enterprise support? Contact us at{' '}
              <a href="mailto:enterprise@surveillanceai.com" className="text-blue-400 hover:text-blue-300 transition-colors">
                enterprise@surveillanceai.com
              </a>
            </p>
          </motion.div>
        </motion.div>
      </div>
    </section>
  );
}
