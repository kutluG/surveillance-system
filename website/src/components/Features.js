import React from 'react';
import {
  CpuChipIcon,
  BellAlertIcon,
  ChartBarIcon,
  ShieldCheckIcon,
  CameraIcon,
  ServerIcon
} from '@heroicons/react/24/outline';

const features = [
  {
    name: 'Real-time Video Processing',
    description: 'Process multiple camera feeds simultaneously with low-latency AI analysis and instant object detection.',
    icon: CameraIcon,
  },
  {
    name: 'AI-Powered Detection',
    description: 'Advanced machine learning algorithms for accurate person, vehicle, and object detection with minimal false positives.',
    icon: CpuChipIcon,
  },
  {
    name: 'Intelligent Alerting',
    description: 'Smart notification system that learns your preferences and reduces alert fatigue while maintaining security.',
    icon: BellAlertIcon,
  },
  {
    name: 'Advanced Analytics',
    description: 'Comprehensive reporting and analytics dashboard with customizable metrics and trend analysis.',
    icon: ChartBarIcon,
  },
  {
    name: 'Enterprise Security',
    description: 'Bank-grade encryption, role-based access control, and compliance with industry security standards.',
    icon: ShieldCheckIcon,
  },
  {
    name: 'Scalable Infrastructure',
    description: 'Cloud-native architecture that scales from single cameras to enterprise deployments of thousands of feeds.',
    icon: ServerIcon,
  }
];

export default function Features() {
  return (
    <div className="py-24 bg-white">
      <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
        <div className="text-center mb-16">
          <h2 className="text-4xl font-bold text-gray-900 mb-4">
            Powerful Features for Modern Surveillance
          </h2>
          <p className="text-xl text-gray-600 max-w-3xl mx-auto">
            Our AI-powered surveillance system combines cutting-edge technology with intuitive design 
            to deliver unparalleled security monitoring capabilities.
          </p>
        </div>

        <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-8">
          {features.map((feature, index) => (
            <div key={feature.name} className="bg-white rounded-lg shadow-lg p-6 border border-gray-100 hover:shadow-xl transition-shadow duration-300">
              <div className="flex items-center mb-4">
                <div className="flex-shrink-0">
                  <feature.icon className="h-8 w-8 text-blue-600" />
                </div>
                <h3 className="ml-3 text-lg font-semibold text-gray-900">{feature.name}</h3>
              </div>
              <p className="text-gray-600">{feature.description}</p>
            </div>
          ))}
        </div>
      </div>
    </div>
  );
}
