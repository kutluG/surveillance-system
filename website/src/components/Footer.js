import React from 'react';
import { Link } from 'react-router-dom';
import { CameraIcon } from '@heroicons/react/24/outline';
import { getServiceUrls } from '../config/urls';

export default function Footer() {
  const serviceUrls = getServiceUrls();
  return (
    <footer className="bg-gray-900 text-white">
      <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8 py-12">
        <div className="grid grid-cols-1 md:grid-cols-4 gap-8">
          {/* Brand */}
          <div className="col-span-1 md:col-span-2">
            <div className="flex items-center space-x-2 mb-4">
              <div className="w-8 h-8 bg-primary-600 rounded-lg flex items-center justify-center">
                <CameraIcon className="w-5 h-5 text-white" />
              </div>
              <span className="text-xl font-bold">SurveillanceAI</span>
            </div>
            <p className="text-gray-300 mb-4 max-w-md">
              Enterprise-grade AI-powered surveillance system with real-time video analysis, 
              intelligent alerting, and comprehensive monitoring capabilities.
            </p>
            <div className="flex space-x-4">
              <a href="https://github.com/kutluG/surveillance-system" 
                 className="text-gray-400 hover:text-white transition-colors">
                <span className="sr-only">GitHub</span>
                <svg className="w-6 h-6" fill="currentColor" viewBox="0 0 24 24">
                  <path fillRule="evenodd" d="M12 2C6.477 2 2 6.484 2 12.017c0 4.425 2.865 8.18 6.839 9.504.5.092.682-.217.682-.483 0-.237-.008-.868-.013-1.703-2.782.605-3.369-1.343-3.369-1.343-.454-1.158-1.11-1.466-1.11-1.466-.908-.62.069-.608.069-.608 1.003.07 1.531 1.032 1.531 1.032.892 1.53 2.341 1.088 2.91.832.092-.647.35-1.088.636-1.338-2.22-.253-4.555-1.113-4.555-4.951 0-1.093.39-1.988 1.029-2.688-.103-.253-.446-1.272.098-2.65 0 0 .84-.27 2.75 1.026A9.564 9.564 0 0112 6.844c.85.004 1.705.115 2.504.337 1.909-1.296 2.747-1.027 2.747-1.027.546 1.379.202 2.398.1 2.651.64.7 1.028 1.595 1.028 2.688 0 3.848-2.339 4.695-4.566 4.943.359.309.678.92.678 1.855 0 1.338-.012 2.419-.012 2.747 0 .268.18.58.688.482A10.019 10.019 0 0022 12.017C22 6.484 17.522 2 12 2z" clipRule="evenodd" />
                </svg>
              </a>
            </div>
          </div>

          {/* Product */}
          <div>
            <h3 className="text-sm font-semibold uppercase tracking-wider mb-4">Product</h3>
            <ul className="space-y-2">
              <li><Link to="/dashboard" className="text-gray-300 hover:text-white transition-colors">Dashboard</Link></li>
              <li><Link to="/cameras" className="text-gray-300 hover:text-white transition-colors">Cameras</Link></li>
              <li><Link to="/alerts" className="text-gray-300 hover:text-white transition-colors">Alerts</Link></li>
              <li><Link to="/analytics" className="text-gray-300 hover:text-white transition-colors">Analytics</Link></li>
            </ul>
          </div>

          {/* Support */}
          <div>
            <h3 className="text-sm font-semibold uppercase tracking-wider mb-4">Support</h3>
            <ul className="space-y-2">
              <li><Link to="/docs" className="text-gray-300 hover:text-white transition-colors">Documentation</Link></li>
              <li><Link to="/pricing" className="text-gray-300 hover:text-white transition-colors">Pricing</Link></li>              <li><a href="mailto:support@surveillanceai.com" className="text-gray-300 hover:text-white transition-colors">Contact</a></li>
              <li><a href={serviceUrls.grafana} target="_blank" rel="noopener noreferrer" className="text-gray-300 hover:text-white transition-colors">Grafana</a></li>
            </ul>
          </div>
        </div>

        <div className="border-t border-gray-800 mt-8 pt-8">
          <div className="flex flex-col md:flex-row justify-between items-center">
            <p className="text-gray-400 text-sm">
              Â© 2025 SurveillanceAI. All rights reserved.
            </p>
            <div className="flex space-x-6 mt-4 md:mt-0">
              <a href="#" className="text-gray-400 hover:text-white text-sm transition-colors">Privacy Policy</a>
              <a href="#" className="text-gray-400 hover:text-white text-sm transition-colors">Terms of Service</a>
            </div>
          </div>
        </div>
      </div>
    </footer>
  );
}
