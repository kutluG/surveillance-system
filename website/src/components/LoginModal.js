import React, { useState } from 'react';
import { motion } from 'framer-motion';
import { Eye, EyeOff, Shield, Lock, User } from 'lucide-react';
import { useApp } from '../contexts/AppContext';

const LoginModal = ({ isOpen, onClose }) => {
  const [credentials, setCredentials] = useState({
    username: '',
    password: ''
  });
  const [showPassword, setShowPassword] = useState(false);
  const [isLoginMode, setIsLoginMode] = useState(true);
  
  const { login, isLoading } = useApp();

  const handleSubmit = async (e) => {
    e.preventDefault();
    
    try {
      await login(credentials);
      onClose();
      setCredentials({ username: '', password: '' });
    } catch (error) {
      console.error('Authentication failed:', error);
    }
  };

  const handleInputChange = (e) => {
    setCredentials({
      ...credentials,
      [e.target.name]: e.target.value
    });
  };

  if (!isOpen) return null;

  return (
    <div className="fixed inset-0 bg-black bg-opacity-50 flex items-center justify-center z-50 p-4">
      <motion.div
        initial={{ opacity: 0, scale: 0.9 }}
        animate={{ opacity: 1, scale: 1 }}
        exit={{ opacity: 0, scale: 0.9 }}
        className="bg-white rounded-xl shadow-2xl max-w-md w-full p-8"
      >
        {/* Header */}
        <div className="text-center mb-8">
          <div className="inline-flex items-center justify-center w-16 h-16 bg-blue-100 rounded-full mb-4">
            <Shield className="h-8 w-8 text-blue-600" />
          </div>
          <h2 className="text-2xl font-bold text-gray-900 mb-2">
            {isLoginMode ? 'Welcome Back' : 'Create Account'}
          </h2>
          <p className="text-gray-600">
            {isLoginMode 
              ? 'Sign in to access your surveillance dashboard' 
              : 'Join our advanced surveillance platform'
            }
          </p>
        </div>

        {/* Form */}
        <form onSubmit={handleSubmit} className="space-y-6">
          {/* Username Field */}
          <div>
            <label htmlFor="username" className="block text-sm font-medium text-gray-700 mb-2">
              Username
            </label>
            <div className="relative">
              <User className="absolute left-3 top-1/2 transform -translate-y-1/2 h-5 w-5 text-gray-400" />
              <input
                type="text"
                id="username"
                name="username"
                value={credentials.username}
                onChange={handleInputChange}
                className="w-full pl-10 pr-4 py-3 border border-gray-300 rounded-lg focus:ring-2 focus:ring-blue-500 focus:border-transparent"
                placeholder="Enter your username"
                required
              />
            </div>
          </div>

          {/* Password Field */}
          <div>
            <label htmlFor="password" className="block text-sm font-medium text-gray-700 mb-2">
              Password
            </label>
            <div className="relative">
              <Lock className="absolute left-3 top-1/2 transform -translate-y-1/2 h-5 w-5 text-gray-400" />
              <input
                type={showPassword ? 'text' : 'password'}
                id="password"
                name="password"
                value={credentials.password}
                onChange={handleInputChange}
                className="w-full pl-10 pr-12 py-3 border border-gray-300 rounded-lg focus:ring-2 focus:ring-blue-500 focus:border-transparent"
                placeholder="Enter your password"
                required
              />
              <button
                type="button"
                onClick={() => setShowPassword(!showPassword)}
                className="absolute right-3 top-1/2 transform -translate-y-1/2 text-gray-400 hover:text-gray-600"
              >
                {showPassword ? <EyeOff className="h-5 w-5" /> : <Eye className="h-5 w-5" />}
              </button>
            </div>
          </div>

          {/* Demo Credentials Note */}
          <div className="bg-blue-50 border border-blue-200 rounded-lg p-4">
            <p className="text-sm text-blue-800 font-medium mb-2">Demo Credentials:</p>
            <p className="text-sm text-blue-700">
              Username: <span className="font-mono">admin</span><br />
              Password: <span className="font-mono">demo123</span>
            </p>
          </div>

          {/* Submit Button */}
          <button
            type="submit"
            disabled={isLoading}
            className="w-full bg-gradient-to-r from-blue-600 to-purple-600 text-white font-semibold py-3 px-4 rounded-lg hover:from-blue-700 hover:to-purple-700 focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2 disabled:opacity-50 disabled:cursor-not-allowed transition-all duration-200"
          >
            {isLoading ? (
              <span className="flex items-center justify-center">
                <div className="animate-spin rounded-full h-5 w-5 border-b-2 border-white mr-2"></div>
                Signing In...
              </span>
            ) : (
              isLoginMode ? 'Sign In' : 'Create Account'
            )}
          </button>
        </form>

        {/* Footer */}
        <div className="mt-6 text-center">
          <button
            type="button"
            onClick={() => setIsLoginMode(!isLoginMode)}
            className="text-blue-600 hover:text-blue-800 font-medium"
          >
            {isLoginMode 
              ? "Don't have an account? Sign up" 
              : "Already have an account? Sign in"
            }
          </button>
        </div>

        {/* Close Button */}
        <button
          onClick={onClose}
          className="absolute top-4 right-4 text-gray-400 hover:text-gray-600"
        >
          <svg className="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
            <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M6 18L18 6M6 6l12 12" />
          </svg>
        </button>
      </motion.div>
    </div>
  );
};

export default LoginModal;
