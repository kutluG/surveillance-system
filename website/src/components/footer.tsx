import Link from "next/link"
import { Shield, Twitter, Github, Linkedin } from "lucide-react"

export function Footer() {
  return (
    <footer className="bg-background border-t border-border/40">
      <div className="mx-auto max-w-7xl px-6 py-12 md:flex md:items-center md:justify-between lg:px-8">
        <div className="flex justify-center space-x-6 md:order-2">
          <Link href="#" className="text-muted-foreground hover:text-foreground">
            <span className="sr-only">Twitter</span>
            <Twitter className="h-6 w-6" />
          </Link>
          <Link href="#" className="text-muted-foreground hover:text-foreground">
            <span className="sr-only">GitHub</span>
            <Github className="h-6 w-6" />
          </Link>
          <Link href="#" className="text-muted-foreground hover:text-foreground">
            <span className="sr-only">LinkedIn</span>
            <Linkedin className="h-6 w-6" />
          </Link>
        </div>
        <div className="mt-8 md:order-1 md:mt-0">
          <div className="flex items-center justify-center md:justify-start">
            <Shield className="h-6 w-6 text-primary mr-2" />
            <span className="text-sm text-muted-foreground">
              Â© 2025 SecureWatch AI. All rights reserved.
            </span>
          </div>
        </div>
      </div>
    </footer>
  )
}
