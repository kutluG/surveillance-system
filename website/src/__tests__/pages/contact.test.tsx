import { render, fireEvent, waitFor } from '@testing-library/react'
import { ThemeProvider } from '@/components/theme-provider'
import ContactPage from '@/app/contact/page'

// Mock next/navigation
jest.mock('next/navigation', () => ({
  useRouter: () => ({
    push: jest.fn(),
    replace: jest.fn(),
    prefetch: jest.fn(),
  }),
  useSearchParams: () => ({
    get: jest.fn(),
  }),
}))

// Mock next-themes
jest.mock('next-themes', () => ({
  useTheme: () => ({
    theme: 'light',
    setTheme: jest.fn(),
  }),
  ThemeProvider: ({ children }: { children: React.ReactNode }) => children,
}))

// Mock fetch for form submission
global.fetch = jest.fn()

const renderWithTheme = (component: React.ReactElement) => {
  return render(
    <ThemeProvider
      attribute="class"
      defaultTheme="system"
      enableSystem
      disableTransitionOnChange
    >
      {component}
    </ThemeProvider>
  )
}

describe('Contact Page', () => {
  beforeEach(() => {
    jest.clearAllMocks()
  })

  it('renders contact page correctly', () => {
    const { container } = renderWithTheme(<ContactPage />)
    expect(container).toMatchSnapshot()
  })

  it('displays contact form', () => {
    const { getByLabelText, getByText } = renderWithTheme(<ContactPage />)
    expect(getByLabelText('Name *')).toBeInTheDocument()
    expect(getByLabelText('Email *')).toBeInTheDocument()
    expect(getByLabelText('Company *')).toBeInTheDocument()
    expect(getByLabelText('Message *')).toBeInTheDocument()
    expect(getByText('Send Message')).toBeInTheDocument()
  })

  it('displays contact information section', () => {
    const { getByText } = renderWithTheme(<ContactPage />)
    expect(getByText('Get in Touch')).toBeInTheDocument()
    expect(getByText('Send us a message')).toBeInTheDocument()
  })

  it('validates form fields', async () => {
    const { getByText, getByRole } = renderWithTheme(<ContactPage />)
    
    const submitButton = getByRole('button', { name: 'Send Message' })
    fireEvent.click(submitButton)
    
    await waitFor(() => {
      expect(getByText('Name must be at least 2 characters')).toBeInTheDocument()
    })
  })

  it('submits form with valid data', async () => {
    const mockResponse = { ok: true, json: async () => ({ success: true }) }
    ;(global.fetch as jest.Mock).mockResolvedValueOnce(mockResponse)

    const { getByLabelText, getByRole } = renderWithTheme(<ContactPage />)
    
    fireEvent.change(getByLabelText('Name *'), { target: { value: 'John Doe' } })
    fireEvent.change(getByLabelText('Email *'), { target: { value: 'john@example.com' } })
    fireEvent.change(getByLabelText('Company *'), { target: { value: 'ACME Corp' } })
    fireEvent.change(getByLabelText('Message *'), { target: { value: 'Test message' } })
    
    const submitButton = getByRole('button', { name: 'Send Message' })
    fireEvent.click(submitButton)
    
    await waitFor(() => {
      expect(global.fetch).toHaveBeenCalledWith('/api/contact', {
        method: 'POST',
        headers: { 'Content-Type': 'application/json' },
        body: JSON.stringify({
          name: 'John Doe',
          email: 'john@example.com',
          company: 'ACME Corp',
          message: 'Test message'
        })
      })
    })
  })
})
