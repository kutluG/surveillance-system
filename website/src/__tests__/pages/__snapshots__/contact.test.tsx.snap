// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Page renders contact page correctly 1`] = `
<div>
  <div
    class="flex flex-col"
  >
    <section
      class="relative bg-gradient-to-b from-background to-muted/20 px-6 py-24 sm:py-32 lg:px-8"
    >
      <div
        class="mx-auto max-w-2xl text-center"
      >
        <h1
          class="text-4xl font-bold tracking-tight text-foreground sm:text-5xl"
        >
          Get in Touch
        </h1>
        <p
          class="mt-6 text-lg leading-8 text-muted-foreground"
        >
          Ready to transform your security infrastructure? Let's discuss how SecureWatch AI can meet your specific needs.
        </p>
      </div>
    </section>
    <section
      class="py-24 sm:py-32"
    >
      <div
        class="mx-auto max-w-7xl px-6 lg:px-8"
      >
        <div
          class="grid grid-cols-1 gap-x-8 gap-y-10 lg:grid-cols-2"
        >
          <div
            class="rounded-lg border bg-card text-card-foreground shadow-sm"
          >
            <div
              class="flex flex-col space-y-1.5 p-6"
            >
              <h3
                class="text-2xl font-semibold leading-none tracking-tight"
              >
                Send us a message
              </h3>
              <p
                class="text-sm text-muted-foreground"
              >
                Fill out the form below and we'll get back to you within 24 hours.
              </p>
            </div>
            <div
              class="p-6 pt-0"
            >
              <form
                class="space-y-6"
              >
                <div
                  class="space-y-2"
                >
                  <label
                    class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
                    for="name"
                  >
                    Name *
                  </label>
                  <input
                    class="flex h-10 w-full rounded-md border border-input bg-background px-3 py-2 text-sm ring-offset-background file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50"
                    id="name"
                    name="name"
                    placeholder="Your full name"
                  />
                </div>
                <div
                  class="space-y-2"
                >
                  <label
                    class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
                    for="email"
                  >
                    Email *
                  </label>
                  <input
                    class="flex h-10 w-full rounded-md border border-input bg-background px-3 py-2 text-sm ring-offset-background file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50"
                    id="email"
                    name="email"
                    placeholder="your.email@company.com"
                    type="email"
                  />
                </div>
                <div
                  class="space-y-2"
                >
                  <label
                    class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
                    for="company"
                  >
                    Company *
                  </label>
                  <input
                    class="flex h-10 w-full rounded-md border border-input bg-background px-3 py-2 text-sm ring-offset-background file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50"
                    id="company"
                    name="company"
                    placeholder="Your company name"
                  />
                </div>
                <div
                  class="space-y-2"
                >
                  <label
                    class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
                    for="message"
                  >
                    Message *
                  </label>
                  <textarea
                    class="flex min-h-[80px] w-full rounded-md border border-input bg-background px-3 py-2 text-sm ring-offset-background placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50"
                    id="message"
                    name="message"
                    placeholder="Tell us about your security requirements, number of cameras, locations, and any specific needs..."
                    rows="5"
                  />
                </div>
                <button
                  class="inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 bg-primary text-primary-foreground hover:bg-primary/90 h-10 px-4 py-2 w-full"
                  type="submit"
                >
                  Send Message
                </button>
              </form>
            </div>
          </div>
          <div
            class="space-y-8"
          >
            <div>
              <h2
                class="text-2xl font-bold tracking-tight text-foreground"
              >
                Contact Information
              </h2>
              <p
                class="mt-4 text-lg text-muted-foreground"
              >
                Prefer to reach out directly? Here are additional ways to get in touch.
              </p>
            </div>
            <div
              class="space-y-6"
            >
              <div
                class="flex items-start space-x-4"
              >
                <div
                  class="flex h-10 w-10 items-center justify-center rounded-lg bg-primary/10"
                >
                  <svg
                    class="lucide lucide-mail h-5 w-5 text-primary"
                    fill="none"
                    height="24"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <rect
                      height="16"
                      rx="2"
                      width="20"
                      x="2"
                      y="4"
                    />
                    <path
                      d="m22 7-8.97 5.7a1.94 1.94 0 0 1-2.06 0L2 7"
                    />
                  </svg>
                </div>
                <div>
                  <h3
                    class="text-lg font-semibold text-foreground"
                  >
                    Email
                  </h3>
                  <p
                    class="text-muted-foreground"
                  >
                    sales@securewatch-ai.com
                  </p>
                  <p
                    class="text-muted-foreground"
                  >
                    support@securewatch-ai.com
                  </p>
                </div>
              </div>
              <div
                class="flex items-start space-x-4"
              >
                <div
                  class="flex h-10 w-10 items-center justify-center rounded-lg bg-primary/10"
                >
                  <svg
                    class="lucide lucide-phone h-5 w-5 text-primary"
                    fill="none"
                    height="24"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M22 16.92v3a2 2 0 0 1-2.18 2 19.79 19.79 0 0 1-8.63-3.07 19.5 19.5 0 0 1-6-6 19.79 19.79 0 0 1-3.07-8.67A2 2 0 0 1 4.11 2h3a2 2 0 0 1 2 1.72 12.84 12.84 0 0 0 .7 2.81 2 2 0 0 1-.45 2.11L8.09 9.91a16 16 0 0 0 6 6l1.27-1.27a2 2 0 0 1 2.11-.45 12.84 12.84 0 0 0 2.81.7A2 2 0 0 1 22 16.92z"
                    />
                  </svg>
                </div>
                <div>
                  <h3
                    class="text-lg font-semibold text-foreground"
                  >
                    Phone
                  </h3>
                  <p
                    class="text-muted-foreground"
                  >
                    +1 (555) 123-4567
                  </p>
                  <p
                    class="text-sm text-muted-foreground"
                  >
                    Monday - Friday, 9AM - 6PM EST
                  </p>
                </div>
              </div>
              <div
                class="flex items-start space-x-4"
              >
                <div
                  class="flex h-10 w-10 items-center justify-center rounded-lg bg-primary/10"
                >
                  <svg
                    class="lucide lucide-map-pin h-5 w-5 text-primary"
                    fill="none"
                    height="24"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M20 10c0 6-8 12-8 12s-8-6-8-12a8 8 0 0 1 16 0Z"
                    />
                    <circle
                      cx="12"
                      cy="10"
                      r="3"
                    />
                  </svg>
                </div>
                <div>
                  <h3
                    class="text-lg font-semibold text-foreground"
                  >
                    Office
                  </h3>
                  <p
                    class="text-muted-foreground"
                  >
                    123 Innovation Drive
                    <br />
                    San Francisco, CA 94105
                    <br />
                    United States
                  </p>
                </div>
              </div>
            </div>
            <div
              class="rounded-lg border text-card-foreground shadow-sm bg-primary/5 border-primary/20"
            >
              <div
                class="flex flex-col space-y-1.5 p-6"
              >
                <h3
                  class="font-semibold tracking-tight text-lg"
                >
                  Enterprise Customers
                </h3>
              </div>
              <div
                class="p-6 pt-0"
              >
                <p
                  class="text-sm text-muted-foreground mb-4"
                >
                  For enterprise inquiries requiring custom solutions, dedicated support, or on-premise deployment, contact our enterprise team directly.
                </p>
                <a
                  class="inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 border border-input bg-background hover:bg-accent hover:text-accent-foreground h-10 px-4 py-2"
                  href="mailto:enterprise@securewatch-ai.com"
                >
                  Contact Enterprise Sales
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section
      class="bg-muted/50 py-24 sm:py-32"
    >
      <div
        class="mx-auto max-w-7xl px-6 lg:px-8"
      >
        <div
          class="mx-auto max-w-2xl text-center"
        >
          <h2
            class="text-3xl font-bold tracking-tight text-foreground sm:text-4xl"
          >
            Common Questions
          </h2>
          <p
            class="mt-4 text-lg text-muted-foreground"
          >
            Quick answers to help you get started
          </p>
        </div>
        <div
          class="mx-auto mt-16 max-w-2xl space-y-8"
        >
          <div>
            <h3
              class="text-lg font-semibold text-foreground"
            >
              How quickly can we get started?
            </h3>
            <p
              class="mt-2 text-muted-foreground"
            >
              Most deployments can begin within 1-2 weeks after initial consultation. The timeline depends on your infrastructure complexity and requirements.
            </p>
          </div>
          <div>
            <h3
              class="text-lg font-semibold text-foreground"
            >
              Do you integrate with existing camera systems?
            </h3>
            <p
              class="mt-2 text-muted-foreground"
            >
              Yes, we support most major camera brands and VMS systems. Our platform can work with your existing infrastructure.
            </p>
          </div>
          <div>
            <h3
              class="text-lg font-semibold text-foreground"
            >
              What about data privacy and compliance?
            </h3>
            <p
              class="mt-2 text-muted-foreground"
            >
              We're fully GDPR compliant with built-in privacy features including face anonymization, encryption, and configurable data retention policies.
            </p>
          </div>
        </div>
      </div>
    </section>
  </div>
</div>
`;
